### ISSUE TITLE
Static Measurement Count

### ISSUE THREAD

**Issue by collina** [m-lab/mlab-vis-client#248]
The number of measurements on the client page seems to be static, and is vastly below our current count. We should update it with the intent of making it a dynamic number at some point.

**Comment by georgiamoon** [comment]
@collina -- do you have a current count that we can update at the moment before we make it dynamic?

**Comment by critzo** [comment]
After review with @georgiamoon @pboothe and @pbeshai we have arrived at an update to the # of tests and # of cities, as well as an agreed upon query for updating these numbers regularly. For # tests, we're using completed, download tests that meet constraints suitable for inclusion in analyses, not all attempted tests, or download tests + upload tests.

**cities: 87000** 
`SELECT count(distinct client_city) FROM [mlab-oti:data_viz.all_ip_by_day]
`

**tests: 242 million** 
```
select count(test_id) as num_tests FROM plx.google:m_lab.ndt.all WHERE IS_EXPLICITLY_DEFINED(web100_log_entry.connection_spec.remote_ip)
AND IS_EXPLICITLY_DEFINED(web100_log_entry.connection_spec.local_ip)
AND IS_EXPLICITLY_DEFINED(web100_log_entry.snap.HCThruOctetsAcked)
AND IS_EXPLICITLY_DEFINED(web100_log_entry.snap.SndLimTimeRwin)
AND IS_EXPLICITLY_DEFINED(web100_log_entry.snap.SndLimTimeCwnd)
AND IS_EXPLICITLY_DEFINED(web100_log_entry.snap.SndLimTimeSnd)
AND project = 0
AND IS_EXPLICITLY_DEFINED(connection_spec.data_direction)
AND connection_spec.data_direction = 1
AND web100_log_entry.snap.HCThruOctetsAcked >= 8192
AND (web100_log_entry.snap.SndLimTimeRwin +
web100_log_entry.snap.SndLimTimeCwnd +
web100_log_entry.snap.SndLimTimeSnd) >= 9000000
AND (web100_log_entry.snap.SndLimTimeRwin +
web100_log_entry.snap.SndLimTimeCwnd +  
web100_log_entry.snap.SndLimTimeSnd) < 3600000000
AND IS_EXPLICITLY_DEFINED(web100_log_entry.snap.CongSignals)
AND web100_log_entry.snap.CongSignals > 0
AND (web100_log_entry.snap.State == 1 OR (web100_log_entry.snap.State >= 5 AND web100_log_entry.snap.State <= 11));
```

**Comment by critzo** [comment]
Addressed in #257
